Added /usgs/sb/ rewrite rules and usgs namespace readme changes (#4042)
perma-id/w3id.org: usgs/README.md

  * sbristol@usgs.gov
  * Github: [https://github.com/skybristol](skybristol)
  * ORCID: [https://orcid.org/0000-0003-1682-4031](0000-0003-1682-4031)

## ScienceBase

> ScienceBase uses a UUID form of identifier that is reasonably permanent as long as ScienceBase continues to exist and operate with its domain and path. Given the possibility of future architecture changes in ScienceBase, the w3id.org rewrite rule can allow for some degree of flexibility in continuing to use the UUID identifiers for an item but redirecting to another infrastructure domain and path in future.

Reviewer: ```suggestion
ScienceBase uses a UUID form of identifier that will be reasonably permanent as long as ScienceBase continues to exist and operate with its domain and path. Given the possibility of future architecture changes in ScienceBase, the w3id.org rewrite rule allows a degree of flexibility in continuing to use the UUID identifiers for an item but redirecting to another infrastructure domain and path in future.
```

### Uses

> This namespace simply includes the UUID identifier under the /usgs/sb/ path and redirects to the item in ScienceBase under /catalog/item/. It was set up for cases where ScienceBase is being used as a long-term archive for items that do not have a DOI and the URLs will be used in and referenced from other third party systems.

Reviewer: ```suggestion
This namespace simply includes the UUID identifier under the `/usgs/sb/` path and redirects to the item in ScienceBase under `/catalog/item/`. It was set up for cases where ScienceBase is being used as a long-term archive for items that do not have a DOI, and the URLs will be used in and referenced from other third party systems.
```
